### DISCUSSION TITLE
form button customization

### DISCUSSION THREAD

**Discussion by asifka** [filamentphp/filament#15018] [General]
Hi,

how to remove create & create another button and want change create button label as well

Thanks

**Comment by leandrocfe** [comment]
If you are using pages, add this to the CreatePage

`protected static bool $canCreateAnother = false;`

If you are using actions

https://filamentphp.com/docs/3.x/actions/prebuilt-actions/create/#disabling-create-another

You can change the label using
`CreateAction::make()->label('xx')`
